lecture/models.py

```
@@ -0,0 +1,24 @@
+from django.db import models
+from classrooms.models import Classroom
+from django.utils.text import slugify
+class Lecture(models.Model):
+    classroom = models.ForeignKey(Classroom, on_delete=models.CASCADE, related_name='lectures')
+    title = models.CharField(max_length=255)
+    pdf_file = models.FileField(upload_to='lecture_pdfs/', blank=True, null=True)  # Single PDF Upload
+    uploaded_on = models.DateTimeField(auto_now_add=True)
+    slug = models.SlugField(max_length=255, unique=True, blank=True)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)